src/scripts/towns/Town.ts

```diff
@@ -753,16 +755,17 @@ const MossdeepAstronomer = new NPC('Astronomer', [
     'There’s a legend that a mythical Wish Pokémon awakens when it passes over us. If you’re as strong as the Champion, maybe you’ll find it roaming around Hoenn granting wishes!',
 ]);
 
+const PacifidlogDiver = new NPC('Diver', [
+    'Yo! Find any cool stuff in chests lately? I\'ve heard that if you beat a Dungeon a lot then the stuff you find in chests gets even more awesome.',
+    'I\'ve even found Masterballs in Chests! Who would put a Masterball in a chest?',
```

Comment:
```suggestion
    'Yo! Find any cool stuff in chests lately?',
    ' I\'ve heard that if you beat a Dungeon a lot then the stuff you find in chests gets even more awesome.',
```
Updated this to remove the Masterballs line, as they're probably going to be removed.